DESCRIPTION

```diff
@@ -26,6 +26,7 @@ Imports:
     flextable,
     ggplot2,
     naniar,
+    nmfspalette,
     stringr,
     tidyr,
     utils
```

Comment:
```suggestion
    utils
  Remotes:
    nmfs-fish-tools/nmfspalette
```

Reply:
@sbreitbart-NOAA you can add remote repositories by adding Remotes: to the description followed by the package path in GitHub. There were some issue with using this in the asar because I added satf as a dependency there, so I am not sure this will 100% work but it should pass the checks. We might need to add that the user needs to download nmfspalette from GitHub in the README